### PR TITLE
Update README with build instruction

### PR DESCRIPTION
**Pull request by lmayencourt**

### CODE REVIEW

**Review by sudpau01 — COMMENTED** [review]
- First it should be pointed to the link https://github.com/ros2-for-arm/ros2/wiki/ROS2-on-arm-architecture in the begining of the page.
- Define the Crosscompile part for the 32 bit toolchain required for compiling the TA

- the file created AMENT IGNORE for not compiling the secure-msg application in https://github.com/ros2-for-arm/ros2/wiki/ROS2-on-arm-architecture should be removed.

**Comment by lmayencourt** [comment]
All the info are now to the wiki. 
The `AMENT_IGNORE` file is not mandatory because the package will depend on optee flags, and not build if optee is not detected.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# example-arm
-ros2 example application specific for arm architecture
+# Build arm examples
+Documentation is in the wiki: https://github.com/ros2-for-arm/ros2/wiki/Secure-messaging-example
```
Comment (david-wang-2015) [inline README.md]: Is the wiki in "ros2" not "example"?

Comment (lmayencourt) [reply README.md]: The only wiki will be the ros2-for-arm/ros2 one. We want to centralise all the information at the same place. All the readme will point to this wiki.

**Review by david-wang-2015 — APPROVED** [review]

**Pull request merged**